packages/dx-react-grid/src/plugins/table-group-header.jsx

```diff
@@ -0,0 +1,109 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import { Getter, Template, Plugin, TemplateConnector, TemplatePlaceholder } from '@devexpress/dx-react-core';
+import {
+  isBandedTableRow,
+  isBandedOrHeaderRow,
+  tableRowsWithBands,
+  isHeadingTableCell,
+  getBandComponent,
+} from '@devexpress/dx-grid-core';
+
+const CellPlaceholder = props => <TemplatePlaceholder params={props} />;
+
+export class TableGroupHeader extends React.PureComponent {
+  render() {
+    const {
+      cellComponent: Cell,
+      rowComponent: HeaderRow,
+      headerCellComponent: HeaderCell,
+      stubCellComponent: StubCell,
+      emptyCellComponent: EmptyCell,
+      columnGroups,
+    } = this.props;
+
+    const tableHeaderRowsComputed = ({ tableHeaderRows }) =>
+      tableRowsWithBands(tableHeaderRows, columnGroups);
+
+    return (
+      <Plugin
+        name="TableGroupHeader"
+        dependencies={[
+          { name: 'Table' },
+          { name: 'TableHeaderRow' },
+          { name: 'DragDropProvider', optional: true },
+          { name: 'TableColumnResizing', optional: true },
+          { name: 'TableSelection', optional: true },
+          { name: 'TableEditColumn', optional: true },
+        ]}
+      >
+        <Getter name="tableHeaderRows" computed={tableHeaderRowsComputed} />
+
+        <Template
+          name="tableCell"
+          predicate={({ tableRow, tableColumn }) => tableRow.type === 'heading' && tableColumn.type === 'flex'}
+        >
+          {params => <StubCell style={{ ...params.style }} rowSpan={params.rowSpan} />}
```

Comment:
Can we use css here? With `:last-of-type` selector, for example?

Reply:
No, because of TableStubCell component will render with `border-left` property in all Grids without Column Groups.
![image](https://user-images.githubusercontent.com/15093892/37832656-4a429dd0-2eba-11e8-8120-f242e8a358f1.png)